Src/FluentAssertions/DataColumnCollectionAssertionExtensions.cs

```diff
@@ -0,0 +1,236 @@
+﻿using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+
+using FluentAssertions.Collections;
+using FluentAssertions.Common;
+using FluentAssertions.Execution;
+
+namespace FluentAssertions
+{
+    public static class DataColumnCollectionAssertionExtensions
+    {
+        /// <summary>
+        /// Asserts that an object reference refers to the exact same object as another object reference.
+        /// </summary>
+        /// <param name="expected">The expected object</param>
+        /// <param name="because">
+        /// A formatted phrase as is supported by <see cref="string.Format(string,object[])" /> explaining why the assertion
+        /// is needed. If the phrase does not start with the word <i>because</i>, it is prepended automatically.
+        /// </param>
+        /// <param name="becauseArgs">
+        /// Zero or more objects to format using the placeholders in <paramref name="because"/>.
+        /// </param>
+        public static AndConstraint<GenericCollectionAssertions<DataColumn>> BeSameAs(this GenericCollectionAssertions<DataColumn> assertion, DataColumnCollection expected, string because = "", params object[] becauseArgs)
```

Comment:
This must be a recent convention, because the identifier `expected` shows up _all over the place_ in other assertions classes, including, evidently, the place I used as a template for this code. :-P I have no issue with changing it though :-)

Reply:
What should I do for the word `unexpected`?

Reply:
Nah, that's why it's a nitpick. And indeed, the code base is inconsistent. Now I think of it, maybe `expectedValue` and `unexpectedValue` is better, because it seems `expectation` doesn't have a clear antonym. So never mind my comment.